_15thnight/templates/index.html

```diff
@@ -3,11 +3,27 @@
 <head>
     <meta charset="UTF-8">
     <meta id="csrf" name="csrf" content="{{ csrf_token() }}">
+    <link href="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+  
+</head>
+    
+	<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<!-- Optional theme -->
+
+
+<!-- Latest compiled and minified JavaScript -->
+<script href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+<link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
+<script src="https://d3js.org/d3.v3.min.js"></script>
```

Comment:
D3 should be used ES6 style with webpack, probably with [this](http://www.reactd3.org/) library.